tests/Fixture/TagsTranslationsFixture.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * CakePHP(tm) Tests <http://book.cakephp.org/2.0/en/development/testing.html>
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://book.cakephp.org/2.0/en/development/testing.html CakePHP(tm) Tests
+ * @since         1.2.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Test\Fixture;
+
+use Cake\TestSuite\Fixture\TestFixture;
+
+/**
+ * Class TagFixture
+ *
+ */
+class TagsTranslationsFixture extends TestFixture
+{
+
+    /**
+     * fields property
+     *
+     * @var array
+     */
+    public $fields = [
+        'id' => ['type' => 'integer', 'null' => false],
+        'locale' => ['type' => 'string', 'null' => false],
+        'name' => ['type' => 'string', 'null' => false],
+        '_constraints' => ['primary' => ['type' => 'primary', 'columns' => ['id', 'locale']]]
+    ];
+
+    /**
+     * records property
+     *
+     * @var array
+     */
+    public $records = [
+        ['id' => '1', 'locale' => 'en_us', 'name' => 'tag 1 translated into en_us'],
```

Comment:
Error says: "**Cannot insert explicit value for identity column** in table 'featured_tags' when IDENTITY_INSERT is set to OFF".  Specifying the 'id' is trying to insert explicit value isn't it?

That said I have no experience with postgres, but have encountered similar error in the past (iirc) and removing ids from records fixed it.